packages/amplify-graphql-transformer-core/API.md

```diff
@@ -768,9 +769,9 @@ export abstract class TransformerPluginBase implements TransformerPluginProvider
 
 // @public (undocumented)
 export class TransformerResolver implements TransformerResolverProvider {
-    constructor(typeName: string, fieldName: string, resolverLogicalId: string, requestMappingTemplate: MappingTemplateProvider, responseMappingTemplate: MappingTemplateProvider, requestSlots: string[], responseSlots: string[], datasource?: DataSourceProvider | undefined);
+    constructor(typeName: string, fieldName: string, resolverLogicalId: string, requestMappingTemplate: MappingTemplateProvider, responseMappingTemplate: MappingTemplateProvider, requestSlots: string[], responseSlots: string[], datasource?: DataSourceProvider | undefined, runtime?: CfnFunctionConfiguration.AppSyncRuntimeProperty | undefined);
```

Comment:
since the `runtime` prop itself is optional, can it just be `CfnFunctionConfiguration.AppSyncRuntimeProperty`?

Reply:
I would think so, but this is autogenerated via `yarn extract-api`; I have no control over it.
Here's the [constructor definition](https://github.com/aws-amplify/amplify-category-api/pull/2812/files#diff-b3129bb53f056565ba11ca27f5846ebb888f0f8bdd643238e2621cd44cd365bdR147) it's based on. You can see the same behavior for the previous property `datasource?`